report/src/problem_analysis/hci/designing_products.tex

```diff
@@ -2,7 +2,7 @@ \section{Designing a product} % generel hci
 When establishing a design for a product, one often begin with exploring initial thoughs and ideas for a design, only to then present the design to the user, make adjustments, and repeat the process. 
 This can be described in process called the design funnel.\cite{BUXTON2007135_skething}
 
-Buxton describes how to move from exploratory analysis of different designs through skething to interactive, product refinement using prototypes.   
+Buxton\cite{BUXTON2007135_skething,BUXTON2007139_prototyping} describes how to move from exploratory analysis of different designs through skething to interactive, product refinement using prototypes.   
```

Comment:
Should use citeauthor instead?

Reply:
This has not been done other places and the formatting is wierd